corehq/apps/app_manager/suite_xml/generator.py

```diff
@@ -134,7 +134,7 @@ def generate_suite(self):
     def media_resources(self):
         PREFIX = 'jr://file/'
         multimedia_map = self.app.multimedia_map_for_build(build_profile=self.build_profile, remove_unused=True)
-        lazy_load_preference = self.app.profile.get('properties', {}).get('lazy-load-video-files')
```

Comment:
Did @shubham1g5 ask for this change? I thought property names were usually dashed case, but he would know for sure.

Reply:
Agree that it should be dashed-case to maintain consistency.

Reply:
Github shows that comment on the old code guess because it was commented on that.
Just in case someone gets confused too, it was changed to `lazy_load_video_files` and hence the comment.

Reply:
So should it be dashed case?

Reply:
right.